### ISSUE TITLE
Is there a pretrained model for the softmax classifier?

### ISSUE THREAD

**Issue by yuvval** [UKPLab/sentence-transformers#301]
First, thank you for sharing your work!

Is there a pretrained model for the softmax classifier? (I.e. for predicting contradiction, entailment, and neutral)
If not, how to set the concatenation for using (u,v, |u-v|) when executing `training_nli.py`?

**Comment by nreimers** [comment]
Hi,
currently not.

If you do not need to pre-compute the sentence embeddings, you might want to use directly huggingface. There, they the BertForSequenceClassification class, which performs exactly this. As it is not based on sentence embeddings, but instead can perform attention across all tokens, it should perform better.

Best
Nils Reimers
